Testing/SystemTests/tests/analysis/LoadVesuvioTest.py

```diff
@@ -20,7 +20,34 @@ def tearDown(self):
         if monitor_name in mtd:
             mtd.remove(monitor_name)
 
+
+    #==================== Test spectrum list validation ============================
+
+    def test_spectrum_list_single_range(self):
+        diff_mode = "DoubleDifference"
+        self._run_load("14188", "10-20", diff_mode)
+
+        # check workspace created
+        self.assertTrue(mtd.doesExist(self.ws_name))
+
+    def test_spectrum_list_comma_separated_list(self):
+        diff_mode = "DoubleDifference"
+        self._run_load("14188", "10,20,30,40", diff_mode)
+
+        # check workspace created
+        self.assertTrue(mtd.doesExist(self.ws_name))
+
+    def test_spectrum_list_comma_separated_ranges(self):
+        diff_mode = "DoubleDifference"
+        self._run_load("14188", "10-20;30-40", diff_mode, do_size_check=False)
```

Comment:
@ElliotAOram As part of the maintenance of the vesuvio tests can you look at adding the size check here too?